apps/website/components/mobile-footer.tsx

```diff
@@ -0,0 +1,57 @@
+import { Facebook, Instagram, Mail, Map, Phone } from "iconoir-react";
+import ExternalLink from "./external-link";
+
+const MobileStickyFooter = () => {
+  return (
+    <footer
+      className="fixed bottom-0 left-0 right-0 h-20 bg-white border-gray-200 md:hidden"
+      data-testid="footer"
+    >
+      <div className="h-full max-w-screen-xl px-4 mx-auto">
+        <div className="flex items-center justify-around h-full">
+          <ExternalLink
+            href="tel:+4939812293575"
+            className="flex  items-center justify-center min-w-[48px] min-h-[48px] text-grau"
+            testId="phone"
+          >
+            <Phone className="w-6 h-6" />
```

Comment:
```suggestion
            <Phone />
```
Die Breiten und Höhenangaben, hatten keine Auswirkung, richtig?